### ISSUE TITLE
2025-05-29, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#184]
# Today's Arxiv Papers

(1) [Zero-Shot Vision Encoder Grafting via LLM Surrogates](http://arxiv.org/abs/2505.22664v1), 2025-05-28

(2) [AutoL2S: Auto Long-Short Reasoning for Efficient Large Language Models](http://arxiv.org/abs/2505.22662v1), 2025-05-28

(3) [GuessArena: Guess Who I Am? A Self-Adaptive Framework for Evaluating LLMs in Domain-Specific Knowledge and Reasoning](http://arxiv.org/abs/2505.22661v1), 2025-05-28

(4) [3DLLM-Mem: Long-Term Spatial-Temporal Memory for Embodied 3D Large Language Model](http://arxiv.org/abs/2505.22657v1), 2025-05-28

(5) [Position: Uncertainty Quantification Needs Reassessment for Large-language Model Agents](http://arxiv.org/abs/2505.22655v1), 2025-05-28

(6) [The Climb Carves Wisdom Deeper Than the Summit: On the Noisy Rewards in Learning to Reason](http://arxiv.org/abs/2505.22653v1), 2025-05-28

(7) [On Learning Verifiers for Chain-of-Thought Reasoning](http://arxiv.org/abs/2505.22650v1), 2025-05-28

(8) [Characterizing Bias: Benchmarking Large Language Models in Simplified versus Traditional Chinese](http://arxiv.org/abs/2505.22645v1), 2025-05-28

(9) [Learning Composable Chains-of-Thought](http://arxiv.org/abs/2505.22635v1), 2025-05-28

(10) [Spatial Knowledge Graph-Guided Multimodal Synthesis](http://arxiv.org/abs/2505.22633v1), 2025-05-28

(11) [Stochastic Chameleons: Irrelevant Context Hallucinations Reveal Class-Based (Mis)Generalization in LLMs](http://arxiv.org/abs/2505.22630v1), 2025-05-28

(12) [Fast-dLLM: Training-free Acceleration of Diffusion LLM by Enabling KV Cache and Parallel Decoding](http://arxiv.org/abs/2505.22618v1), 2025-05-28

(13) [The Entropy Mechanism of Reinforcement Learning for Reasoning Language Models](http://arxiv.org/abs/2505.22617v1), 2025-05-28

(14) [RICO: Improving Accuracy and Completeness in Image Recaptioning via Visual Reconstruction](http://arxiv.org/abs/2505.22613v1), 2025-05-28

(15) [Self-Error-Instruct: Generalizing from Errors for LLMs Mathematical Reasoning](http://arxiv.org/abs/2505.22591v1), 2025-05-28

(16) [Precise In-Parameter Concept Erasure in Large Language Models](http://arxiv.org/abs/2505.22586v1), 2025-05-28

(17) [DocReRank: Single-Page Hard Negative Query Generation for Training Multi-Modal RAG Rerankers](http://arxiv.org/abs/2505.22584v1), 2025-05-28

(18) [Less, but Better: Efficient Multilingual Expansion for LLMs via Layer-wise Mixture-of-Experts](http://arxiv.org/abs/2505.22582v1), 2025-05-28

(19) [Fusion Steering: Prompt-Specific Activation Control](http://arxiv.org/abs/2505.22572v1), 2025-05-28

(20) [Agent-UniRAG: A Trainable Open-Source LLM Agent Framework for Unified Retrieval-Augmented Generation Systems](http://arxiv.org/abs/2505.22571v1), 2025-05-28

(21) [Universal Visuo-Tactile Video Understanding for Embodied Interaction](http://arxiv.org/abs/2505.22566v1), 2025-05-28

(22) [Do Large Language Models Think Like the Brain? Sentence-Level Evidence from fMRI and Hierarchical Embeddings](http://arxiv.org/abs/2505.22563v1), 2025-05-28

(23) [ClaimPKG: Enhancing Claim Verification via Pseudo-Subgraph Generation with Lightweight Specialized LLM](http://arxiv.org/abs/2505.22552v1), 2025-05-28

(24) [DES-LOC: Desynced Low Communication Adaptive Optimizers for Training Foundation Models](http://arxiv.org/abs/2505.22549v1), 2025-05-28

(25) [Emotion-o1: Adaptive Long Reasoning for Emotion Understanding in LLMs](http://arxiv.org/abs/2505.22548v1), 2025-05-28

(26) [A Human-Centric Approach to Explainable AI for Personalized Education](http://arxiv.org/abs/2505.22541v1), 2025-05-28

(27) [PrismLayers: Open Data for High-Quality Multi-Layer Transparent Image Generative Models](http://arxiv.org/abs/2505.22523v1), 2025-05-28

(28) [Multi-MLLM Knowledge Distillation for Out-of-Context News Detection](http://arxiv.org/abs/2505.22517v1), 2025-05-28

(29) [From Strangers to Assistants: Fast Desire Alignment for Embodied Agent-User Adaptation](http://arxiv.org/abs/2505.22503v1), 2025-05-28

(30) [EvolveSearch: An Iterative Self-Evolving Search Agent](http://arxiv.org/abs/2505.22501v1), 2025-05-28

(31) [Topological Structure Learning Should Be A Research Priority for LLM-Based Multi-Agent Systems](http://arxiv.org/abs/2505.22467v1), 2025-05-28

(32) [Fostering Video Reasoning via Next-Event Prediction](http://arxiv.org/abs/2505.22457v1), 2025-05-28

(33) [Unsupervised Post-Training for Multi-Modal LLM Reasoning via GRPO](http://arxiv.org/abs/2505.22453v1), 2025-05-28

(34) [Privacy-preserving Prompt Personalization in Federated Learning for Multimodal Large Language Models](http://arxiv.org/abs/2505.22447v1), 2025-05-28

(35) [Does Johnny Get the Message? Evaluating Cybersecurity Notifications for Everyday Users](http://arxiv.org/abs/2505.22435v1), 2025-05-28

(36) [RAG-Zeval: Towards Robust and Interpretable Evaluation on RAG Responses through End-to-End Rule-Guided Reasoning](http://arxiv.org/abs/2505.22430v1), 2025-05-28

(37) [Zero-Shot 3D Visual Grounding from Vision-Language Models](http://arxiv.org/abs/2505.22429v1), 2025-05-28

(38) [Scaling Reasoning without Attention](http://arxiv.org/abs/2505.22425v1), 2025-05-28

(39) [AI Trust Reshaping Administrative Burdens: Understanding Trust-Burden Dynamics in LLM-Assisted Benefits Systems](http://dx.doi.org/10.1145/3715275.3732077), 2025-05-28

(40) [Self-Reflective Reinforcement Learning for Diffusion-based Image Reasoning Generation](http://arxiv.org/abs/2505.22407v1), 2025-05-28

(41) [Zooming from Context to Cue: Hierarchical Preference Optimization for Multi-Image MLLMs](http://arxiv.org/abs/2505.22396v1), 2025-05-28

(42) [Pangu Embedded: An Efficient Dual-system LLM Reasoner with Metacognition](http://arxiv.org/abs/2505.22375v1), 2025-05-28

(43) [AgentDNS: A Root Domain Naming System for LLM Agents](http://arxiv.org/abs/2505.22368v1), 2025-05-28

(44) [Budget-Adaptive Adapter Tuning in Orthogonal Subspaces for Continual Learning in LLMs](http://arxiv.org/abs/2505.22358v1), 2025-05-28

(45) [LLMs Struggle to Reject False Presuppositions when Misinformation Stakes are High](http://arxiv.org/abs/2505.22354v1), 2025-05-28

(46) [ChatPD: An LLM-driven Paper-Dataset Networking System](http://dx.doi.org/10.1145/3711896.3737202), 2025-05-28

(47) [Advancing Multimodal Reasoning via Reinforcement Learning with Cold Start](http://arxiv.org/abs/2505.22334v1), 2025-05-28

(48) [NLP for Social Good: A Survey of Challenges, Opportunities, and Responsible Deployment](http://arxiv.org/abs/2505.22327v1), 2025-05-28

(49) [Advancing Expert Specialization for Better MoE](http://arxiv.org/abs/2505.22323v1), 2025-05-28

(50) [Chain-of-Thought for Large Language Model-empowered Wireless Communications](http://arxiv.org/abs/2505.22320v1), 2025-05-28

(51) [If Pigs Could Fly... Can LLMs Logically Reason Through Counterfactuals?](http://arxiv.org/abs/2505.22318v1), 2025-05-28

(52) [Skywork Open Reasoner 1 Technical Report](http://arxiv.org/abs/2505.22312v1), 2025-05-28

(53) [From Large AI Models to Agentic AI: A Tutorial on Future Intelligent Communications](http://arxiv.org/abs/2505.22311v1), 2025-05-28

(54) [From Dormant to Deleted: Tamper-Resistant Unlearning Through Weight-Space Regularization](http://arxiv.org/abs/2505.22310v1), 2025-05-28

(55) [CADReview: Automatically Reviewing CAD Programs with Error Detection and Correction](http://arxiv.org/abs/2505.22304v1), 2025-05-28

(56) [Voice CMS: updating the knowledge base of a digital assistant through conversation](http://arxiv.org/abs/2505.22303v1), 2025-05-28

(57) [Adaptive Detoxification: Safeguarding General Capabilities of LLMs through Toxicity-Aware Knowledge Editing](http://arxiv.org/abs/2505.22298v1), 2025-05-28

(58) [Compensating for Data with Reasoning: Low-Resource Machine Translation with LLMs](http://arxiv.org/abs/2505.22293v1), 2025-05-28

(59) [Rethinking the Unsolvable: When In-Context Search Meets Test-Time Scaling](http://arxiv.org/abs/2505.22290v1), 2025-05-28

(60) [Test-Time Immunization: A Universal Defense Framework Against Jailbreaks for (Multimodal) Large Language Models](http://arxiv.org/abs/2505.22271v1), 2025-05-28

(61) [MRT at SemEval-2025 Task 8: Maximizing Recovery from Tables with Multiple Steps](http://arxiv.org/abs/2505.22264v1), 2025-05-28

(62) [Evaluation of LLMs in Speech is Often Flawed: Test Set Contamination in Large Language Models for Speech Recognition](http://arxiv.org/abs/2505.22251v1), 2025-05-28

(63) [BioHopR: A Benchmark for Multi-Hop, Multi-Answer Reasoning in Biomedical Domain](http://arxiv.org/abs/2505.22240v1), 2025-05-28

(64) [Judging Quality Across Languages: A Multilingual Approach to Pretraining Data Filtering with Language Models](http://arxiv.org/abs/2505.22232v1), 2025-05-28

(65) [Look & Mark: Leveraging Radiologist Eye Fixations and Bounding boxes in Multimodal Large Language Models for Chest X-ray Report Generation](http://arxiv.org/abs/2505.22222v1), 2025-05-28

(66) [Investigating Mechanisms for In-Context Vision Language Binding](http://arxiv.org/abs/2505.22200v1), 2025-05-28

(67) [Efficient Leave-one-out Approximation in LLM Multi-agent Debate Based on Introspection](http://arxiv.org/abs/2505.22192v1), 2025-05-28

(68) [Breaking the Cloak! Unveiling Chinese Cloaked Toxicity with Homophone Graph and Toxic Lexicon](http://arxiv.org/abs/2505.22184v1), 2025-05-28

(69) [Speculative Decoding Meets Quantization: Compatibility Evaluation and Hierarchical Framework Design](http://arxiv.org/abs/2505.22179v1), 2025-05-28

(70) [Reverse Preference Optimization for Complex Instruction Following](http://arxiv.org/abs/2505.22172v1), 2025-05-28

(71) [ReliableEval: A Recipe for Stochastic LLM Evaluation via Method of Moments](http://arxiv.org/abs/2505.22169v1), 2025-05-28

(72) [Stratified Selective Sampling for Instruction Tuning with Dedicated Scoring Strategy](http://arxiv.org/abs/2505.22157v1), 2025-05-28

(73) [InComeS: Integrating Compression and Selection Mechanisms into LLMs for Efficient Model Editing](http://arxiv.org/abs/2505.22156v1), 2025-05-28

(74) [What Makes a Good Reasoning Chain? Uncovering Structural Patterns in Long Chain-of-Thought Reasoning](http://arxiv.org/abs/2505.22148v1), 2025-05-28

(75) [EULER: Enhancing the Reasoning Ability of Large Language Models through Error-Induced Learning](http://arxiv.org/abs/2505.22131v1), 2025-05-28

(76) [LoKI: Low-damage Knowledge Implanting of Large Language Models](http://arxiv.org/abs/2505.22120v1), 2025-05-28

(77) [Multilingual vs Crosslingual Retrieval of Fact-Checked Claims: A Tale of Two Approaches](http://arxiv.org/abs/2505.22118v1), 2025-05-28

(78) [THINK-Bench: Evaluating Thinking Efficiency and Chain-of-Thought Quality of Large Reasoning Models](http://arxiv.org/abs/2505.22113v1), 2025-05-28

(79) [Visual Large Language Models Exhibit Human-Level Cognitive Flexibility in the Wisconsin Card Sorting Test](http://arxiv.org/abs/2505.22112v1), 2025-05-28

(80) [Curse of High Dimensionality Issue in Transformer for Long-context Modeling](http://arxiv.org/abs/2505.22107v1), 2025-05-28

(81) [MemOS: An Operating System for Memory-Augmented Generation (MAG) in Large Language Models](http://arxiv.org/abs/2505.22101v1), 2025-05-28

(82) [Knowledge Base Construction for Knowledge-Augmented Text-to-SQL](http://arxiv.org/abs/2505.22096v1), 2025-05-28

(83) [Learning to Route Queries Across Knowledge Bases for Step-wise Retrieval-Augmented Reasoning](http://arxiv.org/abs/2505.22095v1), 2025-05-28

(84) [VIRAL: Vision-grounded Integration for Reward design And Learning](http://arxiv.org/abs/2505.22092v1), 2025-05-28

(85) [iDSE: Navigating Design Space Exploration in High-Level Synthesis Using LLMs](http://arxiv.org/abs/2505.22086v1), 2025-05-28

(86) [ArgInstruct: Specialized Instruction Fine-Tuning for Computational Argumentation](http://arxiv.org/abs/2505.22076v1), 2025-05-28

(87) [Beyond path selection: Better LLMs for Scientific Information Extraction with MimicSFT and Relevance and Rule-induced(R$^2$)GRPO](http://arxiv.org/abs/2505.22068v1), 2025-05-28

(88) [From Failures to Fixes: LLM-Driven Scenario Repair for Self-Evolving Autonomous Driving](http://arxiv.org/abs/2505.22067v1), 2025-05-28

(89) [Weakly Supervised Data Refinement and Flexible Sequence Compression for Efficient Thai LLM-based ASR](http://arxiv.org/abs/2505.22063v1), 2025-05-28

(90) [Safeguarding Privacy of Retrieval Data against Membership Inference Attacks: Is This Query Too Close to Home?](http://arxiv.org/abs/2505.22061v1), 2025-05-28

(91) [Estimating the Effects of Sample Training Orders for Large Language Models without Retraining](http://arxiv.org/abs/2505.22042v1), 2025-05-28

(92) [Detecting Undesired Process Behavior by Means of Retrieval Augmented Generation](http://arxiv.org/abs/2505.22041v1), 2025-05-28

(93) [Jailbreak Distillation: Renewable Safety Benchmarking](http://arxiv.org/abs/2505.22037v1), 2025-05-28

(94) [Analysis and Evaluation of Synthetic Data Generation in Speech Dysfluency Detection](http://arxiv.org/abs/2505.22029v1), 2025-05-28

(95) [CoThink: Token-Efficient Reasoning via Instruct Models Guiding Reasoning Models](http://arxiv.org/abs/2505.22017v1), 2025-05-28

(96) [VulBinLLM: LLM-powered Vulnerability Detection for Stripped Binaries](http://arxiv.org/abs/2505.22010v1), 2025-05-28

(97) [Efficiently Enhancing General Agents With Hierarchical-categorical Memory](http://arxiv.org/abs/2505.22006v1), 2025-05-28

(98) [Leveraging LLM for Stuttering Speech: A Unified Architecture Bridging Recognition and Event Detection](http://arxiv.org/abs/2505.22005v1), 2025-05-28

(99) [Legal Assist AI: Leveraging Transformer-Based Model for Effective Legal Assistance](http://arxiv.org/abs/2505.22003v1), 2025-05-28

(100) [Found in Translation: Measuring Multilingual LLM Consistency as Simple as Translate then Evaluate](http://arxiv.org/abs/2505.21999v1), 2025-05-28
